src/scipp/html/formatting_html.py

```diff
@@ -236,7 +236,7 @@ def find_bin_edges(var, ds):
             # Cannot match dims, just assume length 2 attributes are bin-edge
             if length == 2:
                 bin_edges.append(dim)
-        elif dim in ds.dims and ds.shape[ds.dims.index(dim)] + 1 == length:
+        elif ds.sizes.get(dim, 1) + 1 == length:
```

Comment:
Does this handle bin edges with extra dimensions correctly?

Since this code was written, we have also added https://scipp.github.io/generated/classes/scipp.Coords.html?highlight=is_edges#scipp.Coords.is_edges, I don't know if that helps here.

I also wonder about line 234ff, was this an incomplete solution for the same problem you are fixing here? If so, can it (the `if` branch) be removed, or is it not covered by you solution?

Reply:
> Does this handle bin edges with extra dimensions correctly?

I don't understand what bin edges with extra dimensions means. Can you explain / write a short example?

> I also wonder about line 234ff, was this an incomplete solution for the same problem you are fixing here? 

That might be the case. Ill take a look.

Reply:
Scipp supports "2D" (or higher) coords. This shows up all the time in neutron-scattering data reduction after transforming coords, e.g., from time-of-flight to wavelength, since the transform depends on the other "dimensions" (such as pixel positions). See a concrete example here: https://scipp.github.io/ess/instruments/external/powgen/powgen_reduction.html#Conversion-to-d-spacing

Reply:
I've marked this as a draft because for now I'm not sure what the correct behavior is.
This fix does however do the right thing for the MRE and the example @SimonHeybrock posted in the original issue.